src/components/Login/style.css

```diff
@@ -1,11 +1,25 @@
 .login__google-btn {
   padding: 0.3rem 1rem;
   font-family: 'Mukta';
-  border: none;
-  font-size: medium;
-  border-radius: 5rem;
-  background: rgba(242, 66, 35, 0.928);
-  box-shadow: 2px 4px 5px 0px rgba(115, 115, 115, 0.23);
-  -webkit-box-shadow: 2px 4px 5px 0px rgba(115, 115, 115, 0.23);
-  -moz-box-shadow: 2px 4px 5px 0px rgba(115, 115, 115, 0.23);
+  border: 1px solid rgb(173, 240, 212);
+  font-size: 16px;
+  border-radius: 6px;
+  background: rgb(173, 240, 212);
+}
+.login__google-btn:hover {
+  border: 2px solid rgb(207, 236, 225);
```

Comment:
Kdybys tady nezvetsila o 1px ten border, mozna by ti to cely prihlasovaci tak neposkakovalo.